tests/fixtures/InitCommandTest/migration.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use App\Models\User;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Hash;
+use RonasIT\Support\Traits\MigrationTrait;
+
+class AddDefaultUser extends Migration
+{
+    use MigrationTrait;
+
+    public function up()
+    {
+        if (config('app.env') !== 'testing') {
+            User::create([
```

Comment:
let's use DB facade instead

Reply:
That's test results, which generated by command

Reply:
Done